pkg/common/alert/alert.go

```diff
@@ -82,6 +82,7 @@ func (mas MetricAlerts) Notify() error {
 
 	promAPI := v1.NewAPI(client)
 	for _, ma := range mas {
+		log.Printf("Checking %s", ma.Name)
```

Comment:
Won't hold up the PR on this, but how easy would it be to move this to the metrics client and add the helper functions you need to support it?

Reply:
Agree. :)